src/EditDialog.cpp

```diff
@@ -21,6 +21,591 @@
 #include <QPainter>
 #include <QClipboard>
 
+//  The following section of code should really be in a separate class
+//  but this is only a test!
+
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+#include <ctype.h>
+#include <time.h>
+
+#define INCREMENT 4
+
+#define ERROR_NONE                   0
+#define ERROR_INSUFFICIENT_MEMORY    1
+#define ERROR_INVALID_HEADER         2
+#define ERROR_INVALID_TRAILER        3
+#define ERROR_INVALID_OFFSET         4
+#define ERROR_INVALID_OBJECT_LENGTH  5
+#define ERROR_INVALID_REFERENCE      6
+
+typedef struct KEY {
```

Comment:
`typedef` is unnecessary.

Reply:
Yes, but only if you also change lines 60 and 234. Personally I think typdefs make the code more readable.